Workspaces/Base/Workspace.cs

```diff
@@ -77,11 +83,11 @@ public Bounds outerBounds
 		{
 			get
 			{
-				const float kOuterBoundsCenterOffset = 0.225f; //Amount to lower the center of the outerBounds for better interaction with menus
-				return new Bounds(contentBounds.center + Vector3.down * kOuterBoundsCenterOffset,
+				const float kOuterBoundsCenterOffset = 0.09275f; //Amount to extend the bounds to include frame
```

Comment:
Hm. Should I also change the yBounds const in MiniWorldWorkspace? I realize that we've been inconsistent about const in the local method scope. Some have k, some don't. None of them thus far have had a k_